torchci/lib/bot/pytorchBotHandler.ts

```diff
@@ -102,6 +112,7 @@ class PytorchBotHandler {
     leaveMessage: boolean,
     message: string = "@pytorch bot did not understand your command. Please try `@pytorchbot --help` for other commands."
   ) {
+    await this.logger.log("confused", { message });
```

Comment:
If I'm reading this correctly, this line would result in a record in the table that only contains: 
`confused` : `{ "@pytorch bot did not understand your command..." }`

How would we take that data point and tie it back to specific PRs?

I'm thinking this log is better to leave out

Reply:
The message could be a few different confused messages so we may want data on what type of confusion it was.